tests/e2e/tests/critical-path/browser/formatters.e2e.ts

```diff
@@ -51,137 +36,166 @@ fixture `Formatters`
         await deleteKeysViaCli(keysData);
         await deleteStandaloneDatabaseApi(ossStandaloneConfig);
     });
-test
-    .before(async() => {
-        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
-        // Create new keys
-        await addKeysViaCli(keysData, formatters[0].fromText, formatters[0].fromText);
-    })('Verify that user can see highlighted key details in JSON format', async t => {
-        // Verify for Hash, List, Set, ZSet, String, Stream keys
-        for (let i = 0; i < keysData.length; i++) {
-            const valueSelector = Selector(`[data-testid^=${keysData[i].keyName.split('-')[0]}-][data-testid*=${keysData[i].data}]`);
-            await browserPage.openKeyDetailsByKeyName(keysData[i].keyName);
-            // Verify that json value not formatted with default formatter
-            await t.expect(valueSelector.find(browserPage.cssJsonValue).exists).notOk(`${keysData[i].textType} Value is formatted to JSON`);
-            await browserPage.selectFormatter('JSON');
-            // Verify that json value is formatted and highlighted
-            await t.expect(valueSelector.find(browserPage.cssJsonValue).exists).ok(`${keysData[i].textType} Value is not formatted to JSON`);
-            // Verify that Hash field is formatted to json and highlighted
-            if (keysData[i].keyName === 'hash') {
-                await t.expect(browserPage.hashField.find(browserPage.cssJsonValue).exists).ok('Hash field is not formatted to JSON');
-            }
-            // Verify that Stream field is formatted to json and highlighted
-            if (keysData[i].keyName === 'stream') {
-                await t.expect(Selector(browserPage.cssJsonValue).count).eql(2, 'Hash field is not formatted to JSON');
+formattersHighlightedSet.forEach(formatter => {
```

Comment:
Does it work well?

Reply:
I've used this from testcafe documentation https://testcafe.io/documentation/402804/recipes/best-practices/create-data-driven-tests
and it works perfect